docs/development.rst

```diff
@@ -250,3 +335,29 @@ Publish on Conda Forge
 * send a pull request
 * maintainer will merge if there is no error
 
+
+============================
+Pushing your changes:
+============================
+
+1. Now you have made the changes , tested them, and built them. so now its time to push them.
+
+2. Goto your terminal and type git status and hit enter, this will show your changes from the files
+3. Then type in git add and hit enter, this will add all the files to staging area
+4. Commit the changes by git commit -m "<message-describing-your-change>" and hit enter.
```

Comment:
Highlight the git command and it's parameters with backquotation as code example (backquotes: ``text`` for code samples.)